site/source/docs/building_from_source/building_emscripten_from_source_on_windows.rst

```diff
@@ -24,7 +24,7 @@ Installing required tools
 These instructions explain how to install **all** the :ref:`required tools <toolchain-what-you-need>`. You can :ref:`test whether some of these are already installed <toolchain-test-which-dependencies-are-installed>` on the platform and skip those steps.
 
 
-#. Install `Python 2.x <http://www.python.org/>`_ (not 3.x):
+#. Install `Python 2.7 <http://www.python.org/>`_ (not 3.x):
 
 	- For example `python-2.7.5.amd64.msi <http://python.org/ftp/python/2.7.5/python-2.7.5.amd64.msi>`_ (64-bit version) or `python-2.7.5.msi <http://python.org/ftp/python/2.7.5/python-2.7.5.msi>`_ (32-bit version).
```

Comment:
While you are at it, please also change Python 2.7.5 to Python 2.7.14 in recommendation (basically just bug fixes).

https://www.python.org/downloads/release/python-2714/

Reply:
Maybe 2.7.12? That's what's in the current ubuntu LTS. I don't want people to see 2.7.14 and think they have to get a newer version (unless they have to).

Reply:
I guess Python 2.7.12 is fine too since it was only released on 25 June 2016, though Linux users probably won't read this page.